scraper/scraper.py

```diff
@@ -1,4 +1,4 @@
-from recipe_scrapers import scrape_me
+from recipescrapers.recipe_scrapers import scrape_me
```

Comment:
are you using this file now or not?
plus did you create a requirements.txt file for pip install of all your dependencies?
For running the periodical update we will eventually need one entry point for the AWS Lambda to start from.

Reply:
This file is not in use - I kept it in because I am not sure if @LeshemNoa still needs it.